mne/viz/_brain/_timeviewer.py

```diff
@@ -1199,28 +1200,49 @@ def clean(self):
 class _LinkViewer(object):
     """Class to link multiple _TimeViewer objects."""
 
-    def __init__(self, brains):
+    def __init__(self, brains, time=True, camera=False):
         self.brains = brains
         self.time_viewers = [brain.time_viewer for brain in brains]
 
-        # link time sliders
-        self.link_sliders(
-            name="_time_slider",
-            callback=self.set_time_point,
-            event_type="always"
-        )
+        # check time infos
+        times = [brain._times for brain in brains]
+        if time and not all(len(x) == len(times[0]) for x in times):
+            time = False
+        if time and not all(all(np.equal(x, times[0])) for x in times):
+            time = False
```

Comment:
```suggestion
        if time and not all(np.allclose(x, times[0]) for x in times[1:]):
            warn('stc.times do not match, not linking time')
            time = False
```

Reply:
Why not testing the length?

Reply:
allclose will check shapes implicitly (assuming you don't have something pathological like 1. And [1., 1., 1.], Which would pass allclose but shouldn't occur for data at this level)